core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -199,6 +225,21 @@ public NewTableConfiguration setLocalityGroups(Map<String,Set<Text>> groups) {
     return this;
   }
 
+  /**
+   * Create a new table wkth pre-configured splits from the provided imput collection.
+   *
+   * @param splits
+   *          A SortedSet of String values to be used as split points in a newly created table.
+   * @return this
+   */
+  public NewTableConfiguration withSplits(final SortedSet<Text> splits) {
+    checkArgument(splits != null, "splits set is null");
+    checkArgument(splits.isEmpty() != true, "splits set is empty");
+    this.createInitialSplits = true;
+    this.splitProps = splits;
```

Comment:
Personally I would do `splitProps = ImmuatbleSortedSet.copyOf(splits)` here.  This makes this API's behavior more predictable.